### PR TITLE
[RFC] Kablammo

### PR DESCRIPTION
**Pull request by vivekiitkgp**
@yannickwurm Please review.

Currently, the behavior is as follows:
* Users open the report viewer and is taken to the intro page.
* They select the load file option and select the generated xml file from option.
* Voila!

This commit is based on the changes pending in other PR (which will be finalized soon)

### CODE REVIEW

File: public/ext/kablammo/data/sequenceserver_blast_result20150201-27830-1cjljf8.xml [hunk 1]
```diff
@@ -0,0 +1,2558 @@
+<?xml version="1.0"?>
```
Comment (yeban) [inline public/ext/kablammo/data/sequenceserver_blast_result20150201-27830-1cjljf8.xml]: Is this a sample result or committed by mistake? This needs to be removed. It's confusing.

Comment (vivekiitkgp) [inline public/ext/kablammo/data/sequenceserver_blast_result20150201-27830-1cjljf8.xml]: By mistake.

**Comment by yannickwurm** [comment]
Re the Kablammo report - the user should not have to load/download XML file. A single click to open the report in new window should be plenty. =

**Comment by yeban** [comment]
Also, see if you can't include Kablammo as a git submodule.

**Comment by yannickwurm** [comment]
@yeban  please push to pre make announcement on mailing list without this.

**Comment by yeban** [comment]
@yannickwurm I'm waiting for #129.

**Comment by vivekiitkgp** [comment]
@yannickwurm Kablammo doesn't provide any API as such. For that, I think we need to add custom Javascript to its main page. This will probably prevent us from directly using it as a submodule.

Not sure about it.

**Comment by yeban** [comment]
Worth asking how to go about that on their issue tracker?

**Pull request closed without merging**